xtask/src/themelint.rs

```diff
@@ -157,7 +158,13 @@ pub fn lint(file: String) -> Result<(), DynError> {
         println!("Skipping base16: {}", file);
         return Ok(());
     }
-    let path = path::themes().join(file.clone() + ".toml");
+
+    let path = PathBuf::from(file.clone());
+    let path = if path.is_file() {
+        path
+    } else {
+        path::themes().join(file.clone() + ".toml")
+    };
```

Comment:
Why do you need to do this? Why not just add it to the themes repository?

Reply:
I thought that it would be easier to check a custom theme by just passing its path to the themelint subcommand.
But it's true that I haven't think about just copying the file into the themes repository

Reply:
I guess this isn't an issue, currently themelint needs more work to reduce false positive, but I think we don't really need to allow it to lint custom file, at least I think a better way for this issue is to copy the files to themes repository since sooner or later you will probably submit it to helix anyway.